src/Schema.js

```diff
@@ -326,13 +326,17 @@ class Schema {
     const unresolvedReferences = [];
     this.references.forEach((value, key) => {
       const ref = get(item, key);
+      console.log(key, ref);
+      if (ref === undefined || ref === null) {
```

Comment:
they are mobx observables - not sure if that works then but we'll see i guess

Reply:
they will work, mobx observables return undefined or null :)

Reply:
ah sweet, aproved then